tests/cli/test_compose_live-iso.sh

```diff
@@ -15,40 +15,66 @@ set -e
 CLI="${CLI:-./src/bin/composer-cli}"
 
 rlJournalStart
-    rlPhaseStartSetup
-        rlAssertExists $QEMU_BIN
-    rlPhaseEnd
-
     rlPhaseStartTest "compose start"
         rlAssertEquals "SELinux operates in enforcing mode" "$(getenforce)" "Enforcing"
 
+        TMP_DIR=$(mktemp -d /tmp/composer.XXXXX)
+        SSH_KEY_DIR=$(mktemp -d /tmp/composer-ssh-keys.XXXXXX)
+
+        rlRun -t -c "ssh-keygen -t rsa -N '' -f $SSH_KEY_DIR/id_rsa"
+        PUB_KEY=$(cat "$SSH_KEY_DIR/id_rsa.pub")
+
+        cat > "$TMP_DIR/with-ssh.toml" << __EOF__
+name = "with-ssh"
+description = "HTTP image with SSH"
+version = "0.0.1"
+
+[[packages]]
+name = "httpd"
+version = "*"
+
+[[packages]]
+name = "openssh-server"
+version = "*"
+
+[[packages]]
+name = "beakerlib"
+version = "*"
+
+[customizations.services]
+enabled = ["sshd"]
+
+[[customizations.user]]
+name = "root"
+key = "$PUB_KEY"
+
+[customizations.kernel]
+append = "custom_cmdline_arg"
```

Comment:
You don't assert on this parameter in test_boot_live-iso.sh. Copy&paste error maybe ?